app/controllers/account_requests_controller.rb

```diff
@@ -32,6 +32,7 @@ def create
       redirect_to received_account_requests_path(token: @account_request.identity_token),
                   notice: 'Account request was successfully created.'
     else
+      flash[:alert] = "Name, email, and request details required"
```

Comment:
I'd use the errors of the record to come up with the error string. Maybe right now the only possible error is that one of these three fields is missing, but it might be different in the future.

Reply:
Hi @dorner , great point! 

I have actually removed this flash message because I realized the form had already detailed what is missing/incorrect under each field.